### DISCUSSION TITLE
Ask some ways to translate the terminal interface.

### DISCUSSION THREAD

**Discussion by meBVH** [Malcolm-Q/LC-LateGameUpgrades#465] [General]
With the new UI of the store in the terminal interface , I found that translation mod couldn't work on text about upgrades info.
It is very inconvenient for my friends who are not good at English.
So, are there any ways to translate those in the terminal interface?

**Comment by WhiteSpike** [comment]
Right now, there isn't.
I would have to make it similar with "Override Names". The problem with that is that I would need to tell you the format for each parameter (price, level, additional attributes) so that it shows correctly and some of the descriptions are not formatted the same.
This way, you can put the text to your own language and you can then see them translated in the interface.
